packages/envision/src/scss/pagination.scss

```diff
@@ -44,12 +44,18 @@
 
       @include focus-visible;
 
+      // env-is-active is deprecated
+      &[aria-current='true'],
+      &[aria-current='page'],
```

Comment:
Är det fler vi behöver stödja? T.ex. "step" känns relevant också?

Reply:
Ja, jag kollade på den. Tyckte det var aningen långsökt. Tycker vi lägger till om den efterfrågas :) 
_"the current step within a process such as the current step in an enumerated multi step checkout flow"_